internal/engine/executor.go

```diff
@@ -298,9 +229,7 @@ func (e *Executor) evalEntityEvent(
 func (e *Executor) getEvaluator(
 	ctx context.Context,
 	inf *entities.EntityInfoWrapper,
-	ectx *EntityContext,
```

Comment:
`ectx` was passed in to provide the project ID and the provider name. Since we no longer need provider name, and `inf` has the project ID - get rid of it.